go.mod

```diff
@@ -257,3 +259,5 @@ require (
 	lukechampine.com/blake3 v1.2.1 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
```

Comment:
Note to remove before merging.

Reply:
Just give me a heads up before we merge and I will get it removed in the last commit